File: java/src/kitchen_manager/KitchenManager.java
package kitchen_manager;

public class KitchenManager {
    private Chef chef = new Chef();
    private Dishwasher dishwasher = new Dishwasher();
    private Waiter waiter = new Waiter();

    public void run() {
        chef.prepareFood();
        waiter.serveCustomers();
        dishwasher.washDishes();
    }
}

File: swift/kitchen_manager/KitchenManager.swift
class KitchenManager {
    private let chef = Chef()
    private let dishwasher = Dishwasher()
    private let waiter = Waiter()

    func run() {
        chef.prepareFood()
        waiter.serveCustomers()
        dishwasher.washDishes()
    }
}